key-chord.el

```diff
@@ -110,17 +110,6 @@
 
 ;; ########   Tips   ########################################
 ;;
-;; Don't chord key combinations that exists in the languages you typically
-;; write. Otherwise, if you are typing fast, two key intended to be separate
-;; letters might instead trig a chord.
-;; E.g. "uu" would be a good chord in spanish but not in finnish, and
-;; "hj" would be a good chord in english but not in swedish. 
-;;
-;; Don't rely solely on /usr/dict/words to find unusual combination.
-;; For example "cv" or "fg" can be quite common in certain kinds of
-;; programming. Grep your own texts to verify that a combination is unusual.
-;; And don't forget to check both permutations: "fg" and "gf".
-;;
```

Comment:
Since the previous commit introduced the functional change that makes this tip obsolete (or at least I assume so), you should remove this tip in the same commit.

Reply:
OK, I do.